py/picca/fitter2/utils.py

```diff
@@ -56,11 +63,48 @@ def Pk2Mp(ar,k,pk,ell_max=None):
     return xi
 
 def Pk2Xi(ar,mur,k,pk,ell_max=None):
-    xi=Pk2Mp(ar,k,pk,ell_max)
-    for ell in range(0,ell_max+1,2):
+    ell_vals=[ell for ell in range(0,ell_max+1,2)]
```

Comment:
`range(ell_max+1,2)`

Reply:
range needs three arguments representing start, end, step size.

Reply:
I don't think so. If `start` is not given, it is assumed to be `0`
`range(10)` gives `[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]`.
In the same way, if `step` is not given it is assumed to be `1`.
But you can keep this code, it was only intended as a remark.

Reply:
yes you are right, sorry for the noise.